boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.yaml

```diff
@@ -24,6 +24,7 @@ supported:
   - i2c
   - i2s
   - i3c
+  - nvs
```

Comment:
I am personally OK with using SW features, but Zephyr considers it as a mistake.
Please read the comment of @nashif:
https://github.com/zephyrproject-rtos/zephyr/pull/79924#issuecomment-2422424335

Reply:
Thanks @butok .  But the NVS support does depend on the hardware.  NVS does not support flash with program phrase sizes over 32 Bytes.  This is the major reason this PR moves this SOC to the other driver, to enable NVS.  I feel we do need to filter the NVS tests and samples based on the HW platform.  NXP still has other platforms that are not compatible with NVS, like the LPC5500 family.

Reply:
> But the NVS support does depend on the hardware.

Nice. I will use this argument in future discussions. ;)